static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -794,17 +794,32 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     if (!organization.features.includes('change-alerts')) {
       return this.state.configs?.conditions ?? null;
     }
+    let conditions = this.state.configs?.conditions ?? null;
```

Comment:
i feel like we could check if its null earlier and return early save a bunch of the .? and ?? null repeats

Reply:
i just like writing lots of `???` :)